include/mpsl/mpsl_assert.h

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
Do I need to add rst file for this?

Reply:
@ru-fu Do we?

Reply:
At least, you need to add the API documentation to api.rst.
And ideally we should have conceptual documentation for all libraries as well - seems we don't really have that for MPSL though ...

Reply:
Do you mean nrfxlib/mpsl/doc/api.rst? Because this is in a different repository.

@hurricaneFromMoscow Can we put conceptual documentation for MPSL in the backlog?

Reply:
Yes, I was thinking about nrfxlib - I didn't realize we're in sdk-nrf here.
Then it shouldn't be put in there, but needs an rst file.

Reply:
added rst file